app/models/setting.rb

```diff
@@ -283,19 +283,22 @@ def self.load_defaults
     true
   end
 
+  def self.select_collection_registry
+    @@select_collection ||= SettingSelectCollection.new
```

Comment:
Is there a reason for a class variable instead of instance variable on class object?